sympy/matrices/tests/test_reductions.py

```diff
@@ -1,11 +1,16 @@
 from sympy.matrices.common import _MinimalMatrix, _CastableMatrix
-from sympy.matrices.matrices import MatrixReductions
+from sympy.matrices.matrices import MatrixReductions, _find_reasonable_pivot_naive
 from sympy.testing.pytest import raises
-from sympy.matrices import Matrix
+from sympy.matrices import (
+    GramSchmidt, ImmutableMatrix, ImmutableSparseMatrix, Matrix,
+    SparseMatrix, casoratian, diag, eye, hessian,
+    matrix_multiply_elementwise, ones, randMatrix, rot_axis1, rot_axis2,
+    rot_axis3, wronskian, zeros, MutableDenseMatrix, ImmutableDenseMatrix, MatrixSymbol)
```

Comment:
Please import only the required functions/classes, it may lead to build failure because of unused imports